ember-cli-build.js

```diff
@@ -3,21 +3,22 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function (defaults) {
-  let app = new EmberAddon(defaults, {
-    autoImport: {
```

Comment:
we don't need this, if we include qunit via auto-import

Reply:
I think the reason we had this here was that we weren't sure of broad auto-import support at the time of writing. Makes sense if it's better supported now.